data.go

```diff
@@ -1172,7 +1172,7 @@ func evalHelper(d *Data, env *SymbolTableFrame, needFunction bool) (result *Data
 
 				result, err = Apply(function, args, env)
 				if err != nil {
-					err = errors.New(fmt.Sprintf("\nEvaling %s. %s", String(d), err))
+					err = fmt.Errorf("\nEvaling %s. %w", String(d), err)
```

Comment:
Not sure why this has a newline at the start of the error message, might be worth just removing that.

Reply:
I'll keep it for now so formatting stays the same. It separates nested errors on new lines when printed:

```
Evaling (custom-init). In 'custom-init':
Evaling (initialize). In 'initialize':
Evaling (ignore-error (read (get-pid-...
```